pkg/changelog/unified.go

```diff
@@ -36,7 +36,7 @@ func (c UnifiedChangelog) String() string {
 	return res
 }
 
-// NewUnifiedChangelog creates a unified changelog from varios per-version and
+// NewUnifiedChangelog creates a unified changelog from various per-version and
```

Comment:
Not a blocker but speaking of changelogs, I guess we should probably have a `CHANGELOG.md` in this repo (though I'm guessing we'd need to fix some default output filenames).

Reply:
after convo today, we talked about doing this in a future effort. I logged #153 to track that work.